src/components/ShopDropdownMenu/index.tsx

```diff
@@ -1,20 +1,20 @@
 import { ShopDropdownMenuProps } from '@/types';
 import s from './ShopDropdownMenu.module.scss';
 import { storePanelButtonsList } from '@/constants/storePanelButtonsList';
-import { NavLink } from 'react-router-dom';
+import { Link, NavLink } from 'react-router-dom';
+import { Icon } from '@iconify/react';
 
 export const ShopDropdownMenu = ({ setDropdownOpen }: ShopDropdownMenuProps) => {
 
   
   return (
-    <div className={s.shopDropdown}>
+    <div className={s.shopDropdown} onClick={() => setDropdownOpen(false)}>
       {true ? (
         <div>
           {storePanelButtonsList.map((button) => (
             <NavLink
               to={button.pathToPage}
               key={button.id}
-              onClick={() => setDropdownOpen(false)}
               className={({ isActive }) =>
                 isActive
                   ? `cabinet-sidebar-nav_btn cabinet-sidebar-nav_active`
```

Comment:
These style classes are related to the sidebar, here you should specify those that are related to the dropdown menu